.github/workflows/phpstan.yml

```diff
@@ -19,7 +19,8 @@ jobs:
                 dependencies:
                     - "locked"
                 php-version:
-                    - "7.4"
+                    - "8.0"
+                    - "8.1"
```

Comment:
Same here too, only the lowest version has to be tested.

Reply:
ups.. yes, done :)